### PR TITLE
Adding suggestions on landing page concept for SDKs (i.e - resource links)

### PR DESCRIPTION
**Pull request by jsundai**
## What does this PR do?
Add links to each SDK landing page (index) which includes: 
API docs
SDK github repo
Samples repo
Community forum 
Slack sdk channel

Additionally, adding banner to the dev guides. 

[Preview](https://temporal-documentation-git-landing-page-python-update-concept.preview.thundergun.io/develop/python/)

## Notes to reviewers
EDU-3237
tutorials, links - will update

### CODE REVIEW

File: docs/develop/dotnet/index.mdx [hunk 1]
```diff
@@ -14,7 +14,22 @@ tags:
...
 <!-- .NET SDK feature guidance landing page-->
 
-Build [Temporal Applications](/temporal#temporal-application) with the .Net SDK.
+![.NET SDK Banner](/img/banner-dotnet-temporal.png)
+
+:::info .NET SPECIFIC RESOURCES
+Build [Temporal Applications](/temporal#temporal-application) with the .NET SDK.
+
+**Temporal .NET Technical Resources:**
+- [.NET API Documentation](https://docs.temporal.io/docs/dotnet)
```
Comment (brianmacdonald-temporal) [inline docs/develop/dotnet/index.mdx:23]: https://docs.temporal.io/docs/dotnet doesn't resolve. Should this be https://dotnet.temporal.io/api/ ?

File: docs/develop/dotnet/index.mdx [hunk 2]
```diff
@@ -14,7 +14,22 @@ tags:
...
+- [.NET SDK Code Samples](https://github.com/temporalio/samples-dotnet)
+- [.NET SDK GitHub](https://github.com/temporalio/sdk-dotnet)
+
+**Get Connected with the Temporal .NET Community:**
+
+- [Temporal .NET Community Slack](https://temporalio.slack.com/archives/C012SHMPDDZ)
+- [.NET SDK Forum](https://community.temporal.io/tag/dotnet-sdk)
+- [Temporal In Action: Use Cases with .NET](https://temporal.io/in-use?sdk=Dotnet#temporal-in-action)
```
Comment (brianmacdonald-temporal) [inline docs/develop/dotnet/index.mdx:31]: This link resolves, but there's no content for .NET yet.

File: docs/develop/php/index.mdx [hunk 3]
```diff
@@ -14,8 +14,23 @@ tags:
...
+**Temporal PHP Technical Resources:**
+- [PHP API Documentation](https://php.temporal.io)
+- [PHP SDK Code Samples](https://github.com/temporalio/samples-php)
+- [PHP SDK GitHub](https://github.com/temporalio/sdk-php)
+
+**Get Connected with the Temporal PHP Community:**
+
+- [Temporal PHP Community Slack](https://temporalio.slack.com/archives/C01LK9FAMM0)
+- [PHP SDK Forum](https://community.temporal.io/tag/php-sdk)
+- [Temporal In Action: Use Cases with PHP](https://temporal.io/in-use?sdk=PHP#temporal-in-action)
```
Comment (brianmacdonald-temporal) [inline docs/develop/php/index.mdx:31]: This link also resolves, but there's no content here yet.

File: docs/develop/typescript/index.mdx [hunk 4]
```diff
@@ -14,9 +14,23 @@ tags:
...
+- [TypeScript API Documentation](https://typescript.temporal.io)
+- [TypeScript SDK Code Samples](https://github.com/temporalio/samples-typescript)
+- [TypeScript SDK GitHub](https://github.com/temporalio/sdk-typescript)
+
+**Get Connected with the Temporal TypeScript Community:**
+
+- [Temporal TypeScript Community Slack](https://temporalio.slack.com/archives/C01DKSMU94L)
+- [TypeScript SDK Forum](https://community.temporal.io/tag/typescript-sdk)
+- [Temporal In Action: Use Cases with Typescript](https://temporal.io/in-use?sdk=TypeScript#temporal-in-action)
+- [Temporal Meetups](https://www.meetup.com/pro/temporal-technologies/)
```
Comment (brianmacdonald-temporal) [inline docs/develop/typescript/index.mdx:32]: The Temporal Meetups link only appears on the TypeScript page. Do we want it on all pages?

**Review by brianmacdonald-temporal — COMMENTED** [review]
This looks good for the most part. I noted one broken link, and a couple of others where the links resolve, but there's no content yet. 

I noticed that all the "Temporal In Action" links resolve to the next heading down the page, `#trusted-by-forward-thinking-companies`. Not sure what's causing that, but it's the same in Chrome and Firefox.

File: docs/develop/dotnet/index.mdx (hunk 1 above)
Comment (jsundai) [reply docs/develop/dotnet/index.mdx:23]: Yup! Thank you! 🙌

**Comment by jsundai** [comment]
@brianmacdonald-temporal Feel free to take another look!

**Pull request merged**